conans/client/tools/system_pm.py

```diff
@@ -135,7 +135,7 @@ def install(self, package_name):
         _run(self._runner, "%sapt-get install -y %s%s" % (self._sudo_str, recommends_str, package_name))
 
     def installed(self, package_name):
-        exit_code = self._runner("dpkg -s %s" % package_name, None)
+        exit_code = self._runner("dpkg-query -W -f='${Status}' %s | grep -q \"ok installed\"" % package_name, None)
```

Comment:
What is this `${Status}` about?

Reply:
It's part of a format string which is passed to `dpkg-query`. `-f` is the short form of the `--showformat` command.  http://manpages.ubuntu.com/manpages/cosmic/man1/dpkg-query.1.html

Reply:
Probably I'm dumb but where is the string passed to format the `${Status}`

Reply:
There's no string substitution or formatting happening in python. The `'${Status}'` string gets passed directly to the `dpkg-query` command. At the command line you would use something like:
```bash
dpkg-query -W -f='${Status}' libudev-dev | grep -q "ok installed"
```

Reply:
I'm wondering whether a more portable test would be something like:
```bash
dpkg -l <package-name> 2>/dev/null | grep -q ^ii
```

Reply:
Sorry for the confusion, I was trying to run it in a docker container but wrote up the command incorrectly. I think `dpkg-query -W -f='${Status}' libudev-dev | grep -q "ok installed"` is working well.

Reply:
Note: I confirmed that in a linux with different locale the message is still the same:
French:
```
root@c018a9e1ca3c:/# dpkg-query 
dpkg-query : erreur : requiert une option d'action

Utiliser --help pour de l'aide sur la recherche de paquets.
root@c018a9e1ca3c:/# dpkg-query -W -f='${Status}' zlib1g 
install ok installedroot@c018a9e1ca3c:/# 
```
Russian:
```
root@d947354f90b4:/# dpkg-query 
dpkg-query: ошибка: укажите требуемое действие

Используйте параметр --help для вывода справки по запросам пакетов.
root@d947354f90b4:/# dpkg-query -W -f='${Status}' zlib1g 
install ok installedroot@d947354f90b4:/# 
```